public/javascripts/ViewCarts.js

```diff
@@ -47,12 +47,14 @@ function displayCartInventory() {
     var idSelected = $("#selectDropDown :selected").val();
     $("#qr_button").removeClass("hidden");
 
-    navigation.hit("/Carts/GetCartItems/" + encodeURIComponent(idSelected), function (data) {
-        data = JSON.parse(data);
-        populateCartContainer(data[0]);
-    })/*.fail(function(res) {
-        $("#response").text("Error: displayCartInventory: Connection error.");
-    })*/;
+    navigation.getJSON(window.apiRoute + "/Carts/GetCartItems/" + encodeURIComponent(idSelected), function (err, data) {
```

Comment:
Informational question, What's the difference between navigation.getJSON and doing navigation.get() and then JSON.parse? Should logs be using navigation.getJSON instead?